<?xml version="1.0"?>
<launch>
  <node name="play_piece" pkg="tams_pr2_guzheng" type="play_piece.py" output="screen" />
</launch>